src/utils/cron.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-useless-path-segments */
+const cron = require('node-cron');
+const sendEmail = require('../utils/mailer');
+const logger = require('../utils/logger');
+
+const newsletter = (content) => {
+  cron.schedule(
+    '0 9 * * *',
+    () => {
+      sendEmail(content);
+    },
+    {
+      scheduled: true,
+      timezone: 'Europe/Istanbul',
+    }
+  );
+};
+
+const serverStatus = (content, details) => {
+  cron.schedule(
+    '0 0-23 * * *',
+    () => {
+      logger.info(content, {
+        details,
+      });
+    },
+    {
+      scheduled: true,
+      timezone: 'Europe/Istanbul',
+    }
+  );
+};
```

Comment:
This will schedule a cron task when you call the serverStatus(content, details).
This will send same content every time cron task executes. If you call this function multiple times, it will schedule a new cron task.

You can simply:
```suggestion
  cron.schedule(
    '0 0-23 * * *',
    () => {
      logger.info('Server is up and running'); // If server is not up the task won't execute
      const connection = mongoose.connection
      if(!connection) {
        logger.error(`Db connection not found`)
      } else {
        logger.info(connection)
      }
    },
    {
      scheduled: true,
      timezone: 'Europe/Istanbul',
    }
  );
```

Reply:
All the changes make sense, changed the structure of the app file the way you suggested and changed the newsletter function to send only upcoming projects titles, please keep me in touch if anything else pops into your mind!